docs/source/shared.rst

```diff
@@ -118,7 +121,10 @@ Note: Neither the naive core scheduler nor the multiprocessing scheduler
 are good at workflows with non-trivial cross-task
 communication; they have been removed from the plot.
 
-.. image:: images/scaling-edges.png
+.. figure:: images/scaling-edges.png
+   :alt: Duration in seconds on the y-axis versus number of edges per task on the x-axis. As the number of edges increases from 0 to 100, the time to schedule the entire graph using the threaded scheduler goes from 2 to 8 seconds whereas using the async scheduler goes from 0 to 3 seconds. The cost per edge decreases up until about 10 edges, after which the cost plateaus for both the threaded and async schedulers, with the async scheduler being consistenly faster.
```

Comment:
```suggestion
   :alt: Graph depicting how well Dask scales with the number of edges in the task graph. Graph shows the duration in seconds on the y-axis versus number of edges per task on the x-axis. As the number of edges increases from 0 to 100, the time to schedule the entire graph using the threaded scheduler goes from 2 to 8 seconds whereas using the async scheduler goes from 0 to 3 seconds. The cost per edge decreases up until about 10 edges, after which the cost plateaus for both the threaded and async schedulers, with the async scheduler being consistenly faster.
```

Reply:
Same as above, it's good to start with the big picture before diving into the nitty gritty details.